{"name": "The user should be able to delete rate card sucessfully -- @1.1 ", "status": "broken", "statusDetails": {"message": "smtplib.SMTPAuthenticationError: (534, b'5.7.9 Please log in with your web browser and then try again. For more\\n5.7.9 information, go to\\n5.7.9  https://support.google.com/mail/?p=WebLoginRequired 41be03b00d2f7-6822635db88sm8515009a12.66 - gsmtp')\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 108, in after_step\n    s.login(fromaddr, \"E$$pl@2022!#21\")\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 750, in login\n    raise last_exception\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 739, in login\n    (code, resp) = self.auth(\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 662, in auth\n    raise SMTPAuthenticationError(code, resp)\n"}, "steps": [{"name": "And Enter new \"Rate4\" in Rate Card name field", "status": "passed", "start": 1717050584699, "stop": 1717050586880}, {"name": "And click on save button for rates", "status": "passed", "start": 1717050586881, "stop": 1717050589941}, {"name": "And verify the \"Rate card updated successfully.\"", "status": "broken", "statusDetails": {"message": "smtplib.SMTPAuthenticationError: (534, b'5.7.9 Please log in with your web browser and then try again. For more\\n5.7.9 information, go to\\n5.7.9  https://support.google.com/mail/?p=WebLoginRequired 41be03b00d2f7-6822635db88sm8515009a12.66 - gsmtp')\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 108, in after_step\n    s.login(fromaddr, \"E$$pl@2022!#21\")\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 750, in login\n    raise last_exception\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 739, in login\n    (code, resp) = self.auth(\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 662, in auth\n    raise SMTPAuthenticationError(code, resp)\n"}, "attachments": [{"name": "screenshot", "source": "a6eba4b8-3b5c-46d3-a3ab-7210c6c82420-attachment.png", "type": "image/png"}], "start": 1717050589942, "stop": 1717050603202}, {"name": "And import the rate card file", "status": "skipped", "start": 1717050603205, "stop": 1717050603205}, {"name": "And delete the rate card", "status": "skipped", "start": 1717050603205, "stop": 1717050603205}, {"name": "And verify the \"Rate version(s) deleted successfully.\"", "status": "skipped", "start": 1717050603205, "stop": 1717050603205}], "parameters": [{"name": "Name", "value": "Rate4"}, {"name": "message", "value": "Rate card updated successfully."}, {"name": "message1", "value": "Rate version(s) deleted successfully."}], "start": 1717050583695, "stop": 1717050603205, "uuid": "da93bd89-c616-4846-850e-abb9aa02b980", "historyId": "b4b76b9d87cfccdd81656d092c3df489", "fullName": "Verify the TMS functionality: The user should be able to delete rate card sucessfully", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Rate_Cards_Delete"}, {"name": "feature", "value": "Verify the TMS functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}